src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -242,7 +243,7 @@ trait BeamHelper extends LazyLogging {
       case _ =>
         val (_, outputDirectory) = runBeamWithConfig(config)
         val props = new Properties()
-        props.setProperty("commitHash", LoggingUtil.getCommitHash)
+        props.setProperty("commitHash", getCommitHash)
```

Comment:
This is a bit of personal preference, but I would rather have this be `BashUtil.getCommitHash` as it makes it easier to understand that this logic is being provided elsewhere.